src/adapters/adapter.js

```diff
@@ -144,7 +176,19 @@ class Adapter {
           setTimeout(() => nextChunk(iteration + 1));
         };
 
-        nextChunk();
+        caches.open(CACHE_NAME).then(cache => {
+          globalCache = cache
+          cache.match(CACHE_KEY).then(currentMatch => {
+            if(typeof currentMatch === "undefined" || force){
```

Comment:
Maybe we should check `force` before going into this. That is:

```
if (force || !opts.node /* as commented above, skip cache code if lazy load tree */) nextChunk();
else {
  caches.open()...
}

Reply:
thanks! my bad.